src/main/resources/com/google/api/codegen/py/main.snip

```diff
@@ -211,24 +212,52 @@
                 :func:`google.gax.construct_settings` for the structure of
                 this data. Falls back to the default config if not specified
                 or the specified config is missing data points.
-              app_name (string): The codename of the calling service.
-              app_version (string): The version of the calling service.
+              app_name (string): The name of the application calling
+                the service. Recommended for analytics purposes.
+              app_version (string): The version of the application calling
+                the service. Recommended for analytics purposes.
+              lib_name (string): The API library software used for calling
+                the service. (Unless you are writing an API client itself,
+                leave this as default.)
+              lib_version (string): The API library software version used
+                for calling the service. (Unless you are writing an API client
+                itself, leave this as default.)
+              metrics_headers (dict): A dictionary of values for tracking
```

Comment:
@bjwatson I didn't see mention of this parameter (or allowing generic key-value pairs in the header) in the design doc. Is this how we want to do this in all languages? If not, can the design doc be amended to specify the list of expected configuration parameters available in the GAPIC client library object constructor?